### DISCUSSION TITLE
Monthly Meeting: 3. December 2024, 2pm CET

### DISCUSSION THREAD

**Discussion by KaiVolland** [geostyler/geostyler#2528] [Show and tell]
Participants:
@hwbllmnn @ger-benjamin @jansule @KaiVolland 

## Topics

- Reorganization of geostyler organization on npmjs.com
  - currently there is only one team "developers" so the real contributors of the actual packages are not reflected appropriately
  - reduce amount of people in the geostyler developer group
  - add the contributor image to the repositories --> See TODOs

- CodeSprint
  - maybe around may/june?
  - we stick with 15-20 pax
  - @ger-benjamin will ask if C2C is interested to host it

- Server
  - hetzner server hosted in germany
    - CX32 - VCPUs 4 - RAM 8 - NVME GB 80 - Traffic GB 20 TB
  - terrestris admins will setup a server
  - ~7€ per month --> will be payed by the funding

- FOSSGIS contribution
  - geostyler-lyrx-parser as a lightining talk
  - GeoStyler demo session was rejected :hankey: 

## Todos

- Open thread about CodeSprint 2025 on Discord @KaiVolland 
- Clarify payment via OpenCollective and rent the server @jansule 
- Apply for "OSGeo Project" @KaiVolland create appointment with @jansule 
- Apply for OSGeo Founding (for the CodeSprint) @KaiVolland :point_up: 
- Apply for FOSSGIS Founding (for the CodeSprint)
- Create a geostyler-bot user for the releases --> terrestris admins will create a bot@geostyler.org address
- add the contributor image to the repositories

**Comment by ger-benjamin** [comment]
Hi @KaiVolland, @jansule, @hwbllmnn ,

> CodeSprint - ask if C2C is interested to host it

The answer is positive, we are interested to host it :smiley: 
We fully support the organization of this code sprint. I suppose Bussigny would be the best place for it.
